activity_calendar/templates/activity_calendar/activity_page_base.html

```diff
@@ -153,26 +153,28 @@ <h1>{{ activity_moment.title }}</h1>
                     </div>
                 {% endif %}
 
-                <div class="d-flex">
-                    <!-- Event Participants Icon -->
-                    <div class="col-xs-2">
-                        <img
-                            src="{% static 'octicons/people.svg' %}"
-                            alt=""
-                            class="icon"
-                            height="16"
-                            width="16"
-                        />
-                    </div>
-                    <!-- Patricipants -->
-                    <div class="col nospace align-self-center">
-                        <p id="event-participants">
-                            {{ num_total_participants }} /
-                            {% if num_max_participants == -1 %}&infin;
-                            {% else %}{{ num_max_participants }}{% endif %}
-                        </p>
+                {% if not activity_moment.slot_creation == activity.SLOT_CREATION_NONE %}
```

Comment:
Although `==` has a higher operator precedence than `not` within Python (making this correct in terms of code logic), I'd get rid of the ambiguity
```suggestion
                {% if activity_moment.slot_creation != activity.SLOT_CREATION_NONE %}
```